setup.py

```diff
@@ -31,7 +31,6 @@
     'Programming Language :: Python :: 3.10',
     'Programming Language :: Python :: 3.11',
     'Framework :: Django',
-    'Framework :: Django :: 2.2',
     'Framework :: Django :: 3.0',
```

Comment:
```suggestion
```

Reply:
... same for Django 3.1

Reply:
Done in f5532518775c8c99dbced814676fe834824ae967 :)